apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/Record.java

```diff
@@ -51,4 +83,17 @@ public String key() {
   public Map<String, Object> fields() {
     return fields;
   }
+
+  public int sizeEstimateBytes() {
```

Comment:
This will be used by a bounded memory LRU cache, (and possibly the sql cache -- though SQL might have a better way to track size). In the memory LRU cache, everytime a record is changed (via a merge) we would have to recalculate a size estimate. 

I don't know if I love using the `toJSON` to determine the size, as that isn't very performant, which will be a requirment for the memory cache.

 I'm going to remove the size stuff from this diff, as the serialization for `Record` is valuable in it's own right.

I think for the Memory cache we need very performant